shared/js/background/message-handlers.js

```diff
@@ -33,11 +33,8 @@ export function getBrowser () {
     return browserName
 }
 
-export function firePixel (fireArgs) {
-    if (fireArgs.constructor !== Array) {
-        fireArgs = [fireArgs]
-    }
-    return pixel.fire.apply(null, fireArgs)
+export function submitBrokenSiteReport (brokenSiteArgs) {
+    return brokenSiteReport.fire.apply(null, brokenSiteArgs)
```

Comment:
This is needed to proxy from the UI as a message handler into the broken site report class.

Reply:
There's only one call site and it's always an array so there's no point supporting the other case now.